packages/theming/atlas/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
```

Comment:
This changelog should be for `Atlas Core`

Reply:
Moved it